debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/AbstractMysqlDefaultValueTest.java

```diff
@@ -317,4 +320,36 @@ public void parseNumericAndDecimalToDecimalDefaultValue() {
         assertThat(table.columnWithName("B").defaultValue()).isEqualTo(BigDecimal.valueOf(2.321));
         assertThat(table.columnWithName("C").defaultValue()).isEqualTo(BigDecimal.valueOf(12.678));
     }
+
+    @Test
+    public void parseTimeDefaultValue() {
+        String sql = "CREATE TABLE TIME_TABLE (\n" +
+                "  A timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                "  B timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',\n" +
+                "  C timestamp NOT NULL DEFAULT '0000-00-00 00:00:00.000'," +
+                "  D timestamp NOT NULL DEFAULT '2018-06-26 12:34:56'," +
+                "  E timestamp NOT NULL DEFAULT '2018-06-26 12:34:56.000'," +
+                "  F timestamp NOT NULL DEFAULT '2018-06-26 12:34:56.78'," +
+                "  G datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                "  H datetime NOT NULL DEFAULT '0000-00-00 00:00:00',\n" +
+                "  I datetime(3) NOT NULL DEFAULT '0000-00-00 00:00:00.000'," +
+                "  J datetime NOT NULL DEFAULT '2018-06-26 12:34:56'," +
+                "  K datetime(3) NOT NULL DEFAULT '2018-06-26 12:34:56.000'," +
+                "  L datetime(2) NOT NULL DEFAULT '2018-06-26 12:34:56.78'" +
+                ");";
+        parser.parse(sql, tables);
+        Table table = tables.forTable(new TableId(null, null, "TIME_TABLE"));
+        assertThat(table.columnWithName("A").defaultValue()).isEqualTo("1970-01-01T00:00:00+01:00");
```

Comment:
@jpechane The expected value should be in UTC (i.e. no offset), as per the 0.8 docs:

> The `TIMESTAMP` type represents a timestamp without time zone information and is converted by MySQL from the server (or session's) current time zone into UTC when writing and vice versa when reading back the value.
Such columns are converted into an equivalent `io.debezium.time.ZonedTimestamp` in UTC based on the server (or session's) current time zone.
The timezone will be queried from the server by default.
If this fails, it must be specified explicitly as a connector option using the `database.serverTimezone` option.
So if for instance the database's time zone (either globally or configured for the connector by means of aforementioned option) is "America/Los_Angeles",
the `TIMESTAMP` value "2018-06-20 06:37:03" will be represented by a `ZonedTimestamp` with the value "2018-06-20T13:37:03Z".

Reply:
The expecations for `DATETIME` below look alright.